charts/redisoperator/crds/databases.spotahome.com_redisfailovers.yaml

```diff
@@ -3,11 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    {{- with .Values.crds.annotations }}
-    {{- toYaml . | nindent 4 }}
-    {{- end }}
-    controller-gen.kubebuilder.io/version: (devel)
-    krane.shopify.io/instance-rollout-conditions: "true"
```

Comment:
FWIW,

The CRDs are typically generated via kubebuilder. This particular annotation comes from [kubebuilder annotations on the types in types.go](https://github.com/powerhome/redis-operator/blob/8b0a1a6b022083d36b699c8683d4e090d36b2d8a/api/redisfailover/v1/types.go#L21). 

To further complicate things, the code generator does not managed the helm chart's CRD. The contents here are usually copy-pasta'd from the resource `/manifests`, if I recall. I suspect that' hasn't been done for some time and the current state of the helm chart's CRD is likely woefully behind.